packages/knip/src/plugins/eleventy/index.ts

```diff
@@ -13,14 +16,31 @@ const isEnabled: IsPluginEnabledCallback = ({ dependencies }) => hasDependency(d
 
 const ENTRY_FILE_PATTERNS = ['.eleventy.js', 'eleventy.config.{js,cjs}'];
 
-const PRODUCTION_ENTRY_FILE_PATTERNS = ['posts/**/*.11tydata.js', '_data/**/*.{js,cjs,mjs}'];
+const PRODUCTION_ENTRY_FILE_PATTERNS = ['**/*.11tydata.js', '_data/**/*.{js,cjs,mjs}']
+  .map(pattern => [`src/${pattern}`, `content/${pattern}`])
```

Comment:
Why are `src/` and `content/` added here? They don't seem to be defaults when I look in at https://github.com/11ty/eleventy/blob/main/src/defaultConfig.js but I might miss something?

Reply:
They aren't but the majority of sites I've encountered use that as the source directory (e.g. the official Eleventy blog starter uses `content/`, I and most of the sites I help with in the forum are using `src/`).

Reply:
I guess we can just do the root directory if you want though. I imagine most people will have a configuration file though, so I don't foresee this being used much anyway.

Reply:
Anything that's not default should come from the custom config, i.e. the return value of the `localConfig` function.

By the way, I often run `npm run watch` in the knip repo and then run `knip` from anywhere to test changes. Often that's the fixture folder for the thing I'm workin on. And it's often very insightful to run knip in some project where the code is relevant, in this case such as the ones you mention.